tests/braille-specs/spaces.yaml

```diff
@@ -921,7 +921,7 @@ tests:
   - ['\x200b', '\x0020']
   - ['\x202f', '\x0020']
   - ['\x202f', '\x0020']
-table: he.tbl,spaces.uti
+table: he-IL-comp8.utb,spaces.uti
```

Comment:
This is a nice opportunity to move these tests to he-IL.yaml (and spaces.uti to he-IL-comp8.utb).